src/App.jsx

```diff
@@ -28,6 +30,12 @@ export const App = () => {
             <Route path="/reservations">
               <ReservationPage />
             </Route>
+            <Route path="/rentBikes">
```

Comment:
it is safer not to use camel/pascal case in routes (for HTTP requests), I think. i.e. 'rent-bikes' would be better